### PR TITLE
Handle multipart request

### PR DESCRIPTION
**Pull request by rknanda99**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pw-tech/platform-hyper-express",
-  "version": "0.1.2",
+  "version": "0.1.3",
```
Comment (ezioda004) [inline package.json]: should be a minor version, as its a feature

File: src/adapters/utils/get-body-parser-options.util.ts [hunk 2]
```diff
@@ -28,3 +30,63 @@ export function getBodyParserOptions<Options = NestHyperExpressBodyParserOptions
...
+      const fileStream = field.file.stream;
+
+      // Collect chunks from the file stream
+      for await (const chunk of fileStream) {
+        chunks.push(Buffer.from(chunk));
+      }
+      // Combine all chunks into a single buffer
+      const buffer = Buffer.concat(chunks);
+
+      fields[field.name] = {
```
Comment (ezioda004) [inline src/adapters/utils/get-body-parser-options.util.ts:74]: we can move common object before if/else

File: package.json (hunk 1 above)
Comment (rknanda99) [reply package.json]: done

File: src/adapters/utils/get-body-parser-options.util.ts (hunk 2 above)
Comment (rknanda99) [reply src/adapters/utils/get-body-parser-options.util.ts:74]: done

**Pull request merged**